README.md

```diff
@@ -15,7 +15,11 @@ A simple command handler for [Hikari](https://github.com/hikari-py/hikari).
 ## Features
  - Simple and intuitive API.
  - Slash, user, and message commands.
```

Comment:
perhaps also mention support for autocomplete

Reply:
Autocomplete is mentioned in `Error handling for commands, events, and autocomplete.`. I think it doesn't need to be put someone again.

Reply:
Yeah I noticed that, but it seems a little weird to mention a major feature just in reference to another. Maybe it can be grouped with slash/user/message commands somehow.

Reply:
I thought it wasn't that major since its been out for so long
I'm just gonna put it under "Supports autocomplete'. Sweet and simple.

Reply:
Maybe it's not major, but it's as notable as slash/user/message commands imo.
That works though.